arch/eCos/ua_clock.c

```diff
@@ -24,9 +25,12 @@ UA_Int64 UA_DateTime_localTimeUtcOffset(void){
   return (UA_Int64) (difftime(rawtime, gmt) * UA_DATETIME_SEC);
 }
 
+#define TICKS_TO_MS(Ticks)  \
```

Comment:
This file only exists in the 1.3 branch and before.

We refactored quite a lot of the architecture integration API.
We will bring this back when we can.
But we didn't have the tooling to compile against eCos during the time when the API was revised.